### PR TITLE
Feat/v0

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

File: .github/workflows/CICD.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+on:
+  push:
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+    if: github.ref == 'refs/heads/next' || github.ref == 'refs/heads/master' || github.ref == 'refs/heads/feat/v0'
```
Comment (BotellaA) [inline .github/workflows/CICD.yml:12]:
```suggestion
    if: github.ref == 'refs/heads/next' || github.ref == 'refs/heads/master'
```

File: .github/workflows/CICD.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      - name: Install Node.js, NPM and Yarn
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Install dependencies
+        # if: steps.semantic.outputs.new_release_published == 'true'
+        run: |
+          npm install
+          npm run package
```
Comment (BotellaA) [inline .github/workflows/CICD.yml:37]: do you need package and make and publisher?

File: forge.config.cjs [hunk 3]
```diff
@@ -1,25 +1,29 @@
 module.exports = {
   packagerConfig: {
-    asar: true,
+    asar: false,
```
Comment (BotellaA) [inline forge.config.cjs:3]: if true, does it work?

File: forge.config.cjs [hunk 4]
```diff
@@ -1,25 +1,29 @@
...
+    icon: "./public/favicon.ico",
   },
   rebuildConfig: {},
 
   makers: [
     {
       name: "@electron-forge/maker-zip",
-      config: {},
+      config: {
+        icon: "./public/favicon.ico",
```
Comment (BotellaA) [inline forge.config.cjs:13]: I guess it does not use the packager config???

File: forge.config.cjs [hunk 5]
```diff
@@ -1,25 +1,29 @@
...
       name: "@electron-forge/publisher-github",
       config: {
         repository: {
-          owner: "me",
-          name: "awesome-thing",
+          owner: "Geode-solutions",
+          name: "GeodeApp",
         },
-        prerelease: true,
+        draft: false,
```
Comment (BotellaA) [inline forge.config.cjs:25]: why not prerelese on the next and normal release on master ?

File: nuxt.config.js [hunk 6]
```diff
@@ -1,23 +1,43 @@
...
-        // Main-Process entry file of the Electron App.
         entry: "electron/main.js",
       },
     ],
     renderer: {},
   },
 
+  router: {
+    options: {
+      hashMode: true,
```
Comment (BotellaA) [inline nuxt.config.js:38]: why?

File: pages/index.vue
Comment (BotellaA) [inline pages/index.vue:1]: to remove?

File: nuxt.config.js (hunk 6 above)
Comment (JulienChampagnol) [reply nuxt.config.js:38]: It return 404 when packaged if I remove this

**Pull request closed without merging**